app/views/root/429.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= render partial: "error_page", locals: {
+    title: "Sorry, there was a problem",
```

Comment:
I was wondering that as well. The other pages appear to lack any content design, and I think having a random number is worse for the general, non-technical user.

My suggestion would be to move the status code to a `Status code: <code>` bit at the bottom of the page, as it may be useful for reference.

On the other hand, I'm also trying to avoid changing the existing pages, as that's somewhat out-of-scope. What do you think?

Reply:
Yup, status code at the bottom sounds good. 

Although the number can appear somewhat arbitrary for the user (aside from the ever so familiar 404) they would be useful as something a user quotes if they reference a support problem.

Mostly though I'd be happier if they were handled all in a similar way of all with or without status codes rather than the scatterings both ways - I appreciate that challenge probably started back with decisions made in 2012 though.